packages/labs/context/package.json

```diff
@@ -147,6 +148,7 @@
     "lit": "^2.7.0"
   },
   "devDependencies": {
+    "@esm-bundle/chai": "4.3.4",
```

Comment:
this with `"@esm-bundle/chai": "4.3.4-fix.0"` will give us multiple copies. Can we standardize on one version?

Reply:
Done

Reply:
Done